src/deploy/osp_deployer/settings/sample.properties

```diff
@@ -9,11 +9,11 @@
         "anaconda_iface":"em2.190",
 
         "public_bond": "bond1",
-        "public_slaves": "em2 p1p2",
+        "public_slaves": "em2,p1p2",
```

Comment:
Are these the correct interface names for RHEL8?  I thought they were the ensXXX type of names.

Reply:
Updated the SAH interface names for RHEL8 in sample files. Thank you for pointing it out